### PR TITLE
Prometheus added

### PR DESCRIPTION
**Pull request by beingnishas**
Infrastructure monitoring tool - prometheus is used very prominently by most of the companies as it helps monitor multiple metrics using a single agent called exporter. I think adding prometheus would be apt as this is one of a kind in this list.

### CODE REVIEW

**Comment by SuperQ** [comment]
I'd suggest a slight re-work of that whole "Observability" section to split things up into "Metrics, Logs, Tracing". Then a separate section for SaaS / APM services.

File: contributing.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ Your opinion on value of any resource may not match the opinion of curator.
 ## How are these roadmaps made?
 
 > Roadmaps are made using [Balsamiq](https://balsamiq.com/download/){target=_blank}
+[Relative Link With New Tab](../index.md "Opens new tab for relative home page"){target=_blank}
```
Comment (Ronkiro) [inline contributing.md]: I think you forgot to remove this reference

File: contributing.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ Your opinion on value of any resource may not match the opinion of curator.
 ## How are these roadmaps made?
 
 > Roadmaps are made using [Balsamiq](https://balsamiq.com/download/){target=_blank}
+[Relative Link With New Tab](../index.md "Opens new tab for relative home page"){target=_blank}  
+[Reference Link With New Tab](https://github.com/fire1ce/3os.org/ "Opens new tab for reference link"){target=_blank}
```
Comment (Ronkiro) [inline contributing.md]: Also this one.

**Comment by aleon1220** [comment]
Finally. good work.

File: contributing.md (hunk 2 above)
Comment (beingnishas) [reply contributing.md]: Thanks for letting me know

**Comment by kamranahmedse** [comment]
We can't merge it since the format of the roadmaps doesn't allow conflict resolution easily. However, this has been fixed and will be published in the coming release. Thank you 🙏

**Pull request closed without merging**